extrabees/src/main/java/binnie/extrabees/worldgen/BinnieHiveDescription.java

```diff
@@ -1,8 +1,19 @@
 package binnie.extrabees.worldgen;
 
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
These seem unnecessary?

Reply:
Those were left over from trying to fix another bug, I'll get rid of them.